docs/design.md

```diff
@@ -570,41 +570,25 @@ timestamps such thats<sub>1</sub> \< s<sub>2</sub>.
 
 # Logical Map Content
 
-Logically, the map contains a series of reserved system key / value
-pairs covering accounting, range metadata and node accounting
-before the actual key / value pairs for non-system data
-(e.g. the actual meat of the map).
+Logically, the map contains a series of reserved system key/value
+pairs preceding the actual user data (which is managed by the SQL
+subsystem).
 
-- `\0\0meta1` Range metadata for location of `\0\0meta2`.
-- `\0\0meta1<key1>` Range metadata for location of `\0\0meta2<key1>`.
+- `\x02<key1>`: Range metadata for range ending `\x03<key1>`.
```

Comment:
I think it is worth mentioning this is a `meta1` key. Ditto for the `\x03` and `\x04` prefix below.